### PR TITLE
fix: react-mentions inside react-new-window

### LINKED ISSUE #530: Cannot use react-mentions with react-new-window
**Steps to reproduce:**

1. Render the MentionInput in the NewWindow component from https://www.npmjs.com/package/react-new-window
2. type in the MentionInput 

**Expected behaviour:**
onChange is called

**Observed behaviour:**
onChange is not called

**Workaround:**
None

I have a repository that reproduce the issue here https://github.com/ThibautMarechal/mentions-new-window

It seems to be cause by this check : https://github.com/signavio/react-mentions/blob/master/src/MentionsInput.js#L478-L481

I will a PR that solve my issue by also checking if the navigator is IE to still support that IE bug in the comment. (https://github.com/signavio/react-mentions/issues/14)

### PR DESCRIPTION
**Pull request by ThibautMarechal**
Hi,

Fixes #530

I added a check on the navigator IE to allow react-mentions to be used inside [react-new-window](https://github.com/rmariuzzo/react-new-window)

I'm not sure about how to test that fix. I tested manually but I don't know how to write a test targeting IE and/or another package ? 

It would be great to revalidate the bug #14 to be sure I didn't break anything, but I don't know how I should test that ?

Let me know if I can do anything else about that PR 😉

Thibaut

### CODE REVIEW

File: src/utils/isIE.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const isIE = () => !!document.documentMode
```
Comment (ThibautMarechal) [inline src/utils/isIE.js:1]: From https://stackoverflow.com/a/21712356

**Comment by TheFireCookie** [comment]
Up as this PR would unlock things for my project too!

**Comment by ThibautMarechal** [comment]
Hello @frontendphil, is this project still under maintenance ?

**Comment by frontendphil** [comment]
Hey, @ThibautMarechal I fear not. Both @jfschwarz and I have left Signavio and lost our contributor access to this repo. That means we could also only open up PRs and then not have them merged...

**Comment by ThibautMarechal** [comment]
Hello @atilafassina or @ch-sc, sorry to bother you, but maybe you can help us ? 
I only see you two on github that still work at Signavio.

**Comment by atilafassina** [comment]
> Hello @atilafassina or @ch-sc, sorry to bother you, but maybe you can help us ?
> I only see you two on github that still work at Signavio.

Hello @ThibautMarechal , thank you for the patience and for reaching out.
The short answer is: yes. This repository is still maintained.

It has been abandoned for a few months (as you can probably see by the issues and commits).

The long answer: the governance of this repository has switched hands a few times over the past few months, but we still use it in a few of our apps. So there is interest in keeping lights on over here.


I will have a look at your PR asap, just give me some time to get up to speed with how things are.

**Review by atilafassina — APPROVED** [review]

**Pull request merged**